src/app/clusters/account-login-server/account-login-server.cpp

```diff
@@ -39,18 +39,51 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
 #include <app/util/af.h>
+#include <string>
 
-bool emberAfAccountLoginClusterGetSetupPINCallback(chip::app::Command * commandObj, unsigned char *)
+bool accountLoginClusterIsUserLoggedIn(std::string requestTempAccountIdentifier, std::string requestSetupPin);
+std::string accountLoginClusterGetSetupPin(std::string requestTempAccountIdentifier, chip::EndpointId endpoint);
```

Comment:
Again, putting these sorts of things in `src/app/zap-templates/templates/app/callback.zapt` and `src/app/zap-templates/templates/app/callback-stub-src.zapt` is the way we've done it so far.  Followup for this, please.

Reply:
Will follow up with PR for this.

Reply:
Here's the [issue](https://github.com/project-chip/connectedhomeip/issues/8086) link